cfme/tests/configure/test_zones.py

```diff
@@ -23,25 +23,35 @@ def test_zone_crud(soft_assert):
         casecomponent: WebUI
     """
     zc = current_appliance.collections.zones
+    region = current_appliance.server.zone.region
+
     # CREATE
-    zone = zc.create(
-        name=fauxfactory.gen_alphanumeric(5),
-        description=fauxfactory.gen_alphanumeric(8)
-    )
-    soft_assert(zone.exists, "The zone {} does not exist!".format(
-        zone.description
-    ))
+    name = fauxfactory.gen_alphanumeric(5)
+    description = fauxfactory.gen_alphanumeric(8)
+    zc.create(name=name, description=description)
+
+    # query to get the newly-created zone's id
+    zc.filters = {'parent': region}
+    zones = zc.all()
+    new_zone = None
+    for zone in zones:
+        if (zone.name == name and zone.description == description):
+            new_zone = zone
+            break
+    soft_assert(new_zone is not None and new_zone.exists,
+        "Zone {} could not be created.".format(description))
+
     # UPDATE
-    old_desc = zone.description
-    with update(zone):
-        zone.description = fauxfactory.gen_alphanumeric(8)
-    soft_assert(zone.exists and (old_desc != zone.description),
-                "The zone {} was not updated!".format(
-                    zone.description))
+    with update(new_zone):
+        new_zone.description = fauxfactory.gen_alphanumeric(8)
+    soft_assert(new_zone.exists and (description != new_zone.description),
```

Comment:
Sugegstion :

 Instead of doing this check we can do 

          with update(new_zone):
              new_zone.description = description + "_updated"
              soft_assert(new_zone.exists and (new_zone.description == description + "_updated")

Reply:
Changed.